src/btree/bt_debug.c

```diff
@@ -1502,36 +1543,42 @@ __debug_update(WT_DBG *ds, WT_UPDATE *upd, bool hexbyte)
     for (; upd != NULL; upd = upd->next) {
         switch (upd->type) {
         case WT_UPDATE_INVALID:
-            WT_RET(ds->f(ds, "\tvalue {invalid}\n"));
+            WT_RET(ds->f(ds, "\t\tvalue {invalid}\n"));
```

Comment:
I have added an extra tab, since it looks more clean. Let me know if you guys think otherwise.
Here is what the output looks like:
```
        insert {2,\dd\a3,0,0}
                value {tombstone}
                txn id 17656, start_ts (0, 0)
                value {0,0,3,aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa}
```

Reply:
I agree the debug functions aren't terribly intuitive in their output (it's hard to tease apart key/value timestamps and items in some of the output), but I am hesitant to make this change, unless we were consistent about it for all page formats, always indenting the value. (I also don't like the change much, but I wouldn't fight about it too hard.)

Reply:
I have reverted the change as well!